lib/Settings/Settings.cpp

```diff
@@ -31,34 +31,51 @@ void Settings::deserialize(Settings& settings, JsonObject& parsedSettings) {
     if (parsedSettings.containsKey("admin_username")) {
       settings.adminUsername = parsedSettings.get<String>("admin_username");
     }
-    
+
     if (parsedSettings.containsKey("admin_password")) {
       settings.adminPassword = parsedSettings.get<String>("admin_password");
     }
-    
+
     if (parsedSettings.containsKey("ce_pin")) {
       settings.cePin = parsedSettings["ce_pin"];
     }
-    
+
     if (parsedSettings.containsKey("csn_pin")) {
       settings.csnPin = parsedSettings["csn_pin"];
     }
-    
+
+    if (parsedSettings.containsKey("reset_pin")) {
+      settings.resetPin = parsedSettings["reset_pin"];
+    }
+
+    if (parsedSettings.containsKey("radio_interface_type"))
+    {
+      String InterfaceType = parsedSettings["radio_interface_type"];
+      if(InterfaceType == "nRF24")
+      {
+        settings.radioInterfaceType =  nRF24;
+      }
+      else if(InterfaceType == "PL1167/LT8900/8910/8920")
```

Comment:
Does this thing have a standard name? Is one of these the "real" name and others are resold under different model numbers? I think it's fine to use this name to display in the UI, but I'm thinking it should probably have a tighter representation in code.

Might be nice to pull this into a helper since it's used more than once.